subsys/bootloader/Kconfig

```diff
@@ -105,18 +105,40 @@ config SB_DEBUG_SIGNATURE_PUBLIC_KEY_LAST
 	  first. This is meant to be used for testing looping through the
 	  public keys.
 
-endif # SECURE_BOOT
-
-menuconfig IS_SECURE_BOOTLOADER
-	bool "Current app is bootloader"
-	depends on SECURE_BOOT_VALIDATION
-	select SW_VECTOR_RELAY if SOC_SERIES_NRF51X
-	select NRFX_NVMC
+config SB_MONOTONIC_COUNTER
+	bool "Enable HW monotonic version counter"
+	default y
 	help
-	  This option is set by the first stage bootloader app to include all
-	  files and set all the options required.
+	  An array of slots is used for one counter. One slot is used per
+	  counter update, and each new slot must have a larger value than the
+	  previous. Application versions are checked against the current largest
+	  counter before being booted.
+
+if SB_MONOTONIC_COUNTER
+config SB_NUM_MONOTONIC_COUNTERS
```

Comment:
Should we use `depends on` instead? To me that is more readable as you don't have to 'remember' the context around the kconfig.

Reply:
np, they are equivalent, changed now.